dev/integration_tests/web_e2e_tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Flutter for Web Integration testing with `integration_test` package
+
+These are web tests which are using `integration_test` (aka e2e) package. In order to run them locally, run `chromedriver`:
+
+```
+./chromedriver --port=4444
```

Comment:
This assumes that you're in the dir where chromedriver is downloaded, I have added it to the path so I don't need the `./` to run chromedriver.

(Adding Chromedriver to the path is described in their [Getting Started](https://chromedriver.chromium.org/getting-started) guide)

Reply:
done. thanks for the link!